client/layout/masterbar/logged-in.jsx

```diff
@@ -287,6 +289,16 @@ class MasterbarLoggedIn extends React.Component {
 							{ translate( 'Write' ) }
 						</AsyncLoad>
 					) }
+					{ shouldShowCartIcon && (
+						<AsyncLoad
+							require="./masterbar-cart"
+							placeholder={ null }
+							className="masterbar__item-cart"
+							tooltip={ translate( 'View my Shopping Cart' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **34** times:
`translate( 'My Shopping Cart' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/masterbar-cart)